src/components/organisms/PostCollection/PostCollection.jsx

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+import { Post } from '../..';
+
+export const PostCollection = ({ userPosts }) => {
+    return userPosts.map(({ postID }) => <Post postID={postID} />);
```

Comment:
~~`{ postID }`~~

`postID`

Reply:
line 5

Reply:
we pass an array of strings, not an array of { postID: string } objects

Reply:
small typo, everything seems fine, thanks!

Reply:
Done